test/integration/gardenlet/seed/seed/seed_test.go

```diff
@@ -590,6 +593,9 @@ var _ = Describe("Seed controller tests", func() {
 											End:   "230000+0100",
 										},
 									},
+									Networking: operatorv1alpha1.Networking{
+										Services: "100.64.0.0/13",
```

Comment:
Do we want to add some validation for the service cidr, i.e. check that it is actually a cidr? At least in this commit I did not find any.

Reply:
Will do